db/sql/init_db.sql

```diff
@@ -37,4 +37,59 @@ CREATE TABLE IF NOT EXISTS credentials (
     created_at TIMESTAMP DEFAULT NOW(),
     updated_at TIMESTAMP DEFAULT NOW(),
     deleted_at TIMESTAMP
-);
\ No newline at end of file
+);
+
+-- COURSES TABLE
+CREATE TABLE IF NOT EXISTS courses (
+    id SERIAL PRIMARY KEY,
+    course_code VARCHAR NOT NULL,          
+    course_name VARCHAR NOT NULL,          
+    term VARCHAR NOT NULL,                 
+    section VARCHAR,                       
+    instructor_id INTEGER NOT NULL REFERENCES users(id) ON DELETE CASCADE,
```

Comment:
This way of storing professor will only allow for one professor per course. I think we should just have professors information also stored in enrollments table.